python/core/auto_generated/qgsapplication.sip.in

```diff
@@ -378,7 +378,7 @@ colors are specified then the ``size`` argument also must be set.
 Returns the path to user's style.
 %End
 
-    static QRegExp shortNameRegExp();
+    static QRegularExpression shortNameRegExp();
```

Comment:
You'll need to revert anything which changes the Python bindings -- because these are part of the stable API and QRegularExpression isn't api compatible with QRegEx, we risk breaking existing python plugins and scripts if we alter the types now.

Reply:
Sure thing!

Reply:
I have a question regarding the best approach to this, for functions that require regex input, should I duplicate them and convert the regex to regulareexpression to call the new version or should I not do any conversion and leave untouched anything that would need a regexp in the API ( this might be a lot of stuff to revert).

Reply:
Api should be safe now.